dspace-xmlui/src/main/webapp/themes/UFAL/lib/js/messages/messages_sl.js

```diff
@@ -0,0 +1,29 @@
+messages = {
+    "autocomplete-funding-code": "Koda projekta:",
+    "autocomplete-project-name": "Ime projekta:",
+    "autocomplete-used-by-submission": "se uporablja v %s vnosih",
+    "autocomplete-iso": "Koda",
+    "autocomplete-language": "Jezik",
+    "autocomplete-used-by": "se uporablja %s&times;",
+    "autocomplete-original-data-help": "Najnovejše dostopne podatke najdete v zgornjem vnosu. Če vseeno potrebujete izvorne podatke, prosim kliknite  ",
+    "autocomplete-here": "sem",
+    "autocomplete-cookie-help": "Ta stran uporablja piškotke. Z nadaljevanjem uporabe te strani soglašate z uporabo piškotkov.",
+    "autocomplete-cookie-understand": "Razumem",
+    "autocomplete-cookie-more-info": "Več informacij",
+    "dragndrop-upload-failed": "Ne morem prevzeti teh datotek.",
+    "dragndrop-drop-here": "Spustite datoteko/e sem.",
+    "upload-rejected-files": "Zavrnjene datoteke",
+    "upload-toolarge-help": "The following files are too large for conventional upload (limit is %s) or are empty (0 bytes). Please contact %s about how to upload these files.",
+    "upload-filename": "Ime datoteke",
+    "upload-size": "Velikost",
+    "upload-file-upload": "Prenos datoteke",
+    "upload-fill-description": "Izpolnite prosim opis(e) in kliknite na \"Začni prenos\".\nPtem počakajte, da se prenesejo vse datoteke.",
+    "upload-start-upload": "Začni prenos",
+    "upload-describe-file": "Opišite datoteke",
+    "upload-progress": "Napredek",
+    "upload-upload-failed": "Prenos %s ni uspel.\n%s\n%s",
+    "upload-type": "Tip",
+    "upload-done": "Končano"
+};
+
+jQuery.i18n.load('sl', messages);
```

Comment:
@kosarko Yes, I wasn't sure what was the right thing to do here. The left-out keys are only used in error.html, but we decided not to promote upstream inclusion of localisation of the static html files. My inclination is thus to include the static-only keys along with the static html files only in our fork. Does that seem reasonable? Whatever we decide, I should probably mention something about this in the wiki.